helm/flux-app/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.19.0
+appVersion: 0.20.1
```

Comment:
I think we need to bump the version via kustomize. See https://github.com/giantswarm/flux-app/pull/50

The upstream YAML is defined here https://github.com/giantswarm/flux-app/blob/2a63d57608e1943ae671196f3e2952d8deceec38/hack/kustomization.yaml#L27

Reply:
Thanks, done!